src/commands/miscellaneous/Feedback.ts

```diff
@@ -29,14 +32,20 @@ export default class FeedbackCommand extends BaseCommand {
 		client: DiscordClient<true>,
 		interaction: DiscordChatInputCommandInteraction<"cached">
 	) {
-		const guildPreferences = await GuildPreferencesCache.get(
-			interaction.guildId
-		);
 
-		if (!guildPreferences || !guildPreferences.feedbackChannelId) {
+		const feedbackTeams = await FeedbackChannels.find({
+			$or: [{
+				guildId: interaction.guildId
+			}, {
+				label: "Bot Developers",
+				channelId: process.env.DEV_FEEDBACK_CHANNEL_ID
+			}]
+		});
+
+		if (feedbackTeams.length == 0) {
 			await interaction.reply({
 				content:
-					"Please setup the bot using the command `/setup` first.",
+					"Please setup feedback channels using the command `/feedback_channel add` first.",
```

Comment:
Fix this message. Just say "No feedback teams" or something similar. And there should always be 1 feedback team, us bot devs should always be visible so remove this altogether